infrastructure/zk/src/status.ts

```diff
@@ -0,0 +1,210 @@
+import { Command } from 'commander';
+
+import { Pool } from 'pg';
+import { ethers } from 'ethers';
+import { assert } from 'console';
+
+// Postgress connection pool - must be intialized later - as the ENV variables are set later.
+let pool: any = null;
```

Comment:
To still get type hints, you can do something like
```suggestion
let pool: Pool | null = null;
```

Reply:
done.